src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/kgdViscosityDominated/kgdViscosityDominatedQueries.py

```diff
@@ -0,0 +1,116 @@
+import sys
+import matplotlib
+import numpy as np
+import matplotlib.pyplot as plt
+import math
+from math import sin,cos,tan,exp
+import h5py
+import xml.etree.ElementTree as ElementTree
+
+
+def getMeshSettings(xmlFilePath):
+    
+    tree = ElementTree.parse(xmlFilePath)
+    meshElement = tree.find('Mesh/InternalMesh')
+
+    nodeXCoords = meshElement.get("xCoords")
+    nodeYCoords = meshElement.get("yCoords")
+    nodeZCoords = meshElement.get("zCoords")
+    
+    nodeXCoords = [float(i) for i in nodeXCoords[1:-1].split(",")]
+    nodeYCoords = [float(i) for i in nodeYCoords[1:-1].split(",")]
+    nodeZCoords = [float(i) for i in nodeZCoords[1:-1].split(",")]    
+
+    xMin = nodeXCoords[0]
+    xMax = nodeXCoords[-1]
+    yMin = nodeYCoords[0]
+    yMax = nodeYCoords[-1]
+    zMin = nodeZCoords[0]
+    zMax = nodeZCoords[-1]
+
+    nXElem = meshElement.get("nx")
+    nYElem = meshElement.get("ny")
+    nZElem = meshElement.get("nz")
+    nXElem = [float(i) for i in nXElem[1:-1].split(",")]
+    nYElem = [float(i) for i in nYElem[1:-1].split(",")]
+    nZElem = [float(i) for i in nZElem[1:-1].split(",")]  
+    nx = nXElem[0]
+    ny = nYElem[0]
+    nz = nZElem[0]
+    
+    return xMin, xMax, yMin, yMax, zMin, zMax, nx, ny, nz
+
+def SaveTimeHistory( name, array, header ):
```

Comment:
Since this is a user-facing example, could you create a docstring that defines the function IO?  I tend to use the google style elsewhere in the code (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)

Reply:
delete this function and use `np.savetxt` as suggested